packages/mrgn-ui/src/components/ui/icons.tsx

```diff
@@ -1,5 +1,5 @@
 import { cn } from "@mrgnlabs/mrgn-utils";
-import { IconLoader2 } from "@tabler/icons-react";
+import { IconLoader2, IconInfoCircle as _IconInfoCircle } from "@tabler/icons-react";
```

Comment:
we're moving away from having our icons file and importing directly from @tabler

Reply:
Okay, I'll leave it as is for now and once we do the big refactor of our icons we'll handle this